test/Ocelot.IntegrationTests/AdministrationTests.cs

```diff
@@ -878,11 +878,11 @@ private void ThenTheStatusCodeShouldBe(HttpStatusCode expectedHttpStatusCode)
         {
             _response.StatusCode.ShouldBe(expectedHttpStatusCode);
         }
-        
+
         private void ThenTheResultHaveMultiLineIndentedJson()
         {
             const string indent = "  ";
-            const int total = 46, skip = 1;
+            const int total = 47, skip = 1;
```

Comment:
Is this test still failing?
Seems the test is stabilized on the line # 887

Reply:
ok